app/src/behaviors/behavior_turbo_key.c

```diff
@@ -0,0 +1,156 @@
+    data->release_at = event.timestamp + data->wait_ms;
+    int32_t ms_left = data->release_at - k_uptime_get();
+    if (ms_left > 0) {
+        k_work_schedule(&data->release_timer, K_MSEC(ms_left));
+        LOG_DBG("Successfully reset turbo timer at position %d", data->position);
+    }
+}
+
+static void behavior_turbo_timer_handler(struct k_work *item) {
+    struct behavior_turbo_data *data =
+        CONTAINER_OF(item, struct behavior_turbo_data, release_timer);
+    if (!data->is_active) {
+        return;
+    }
+    if (data->timer_cancelled) {
+        return;
+    }
+    LOG_DBG("Turbo timer reached.");
+    struct zmk_behavior_binding_event event = {.position = data->position,
+                                               .timestamp = k_uptime_get()};
+    zmk_behavior_queue_add(event.position, data->binding, true, data->tap_ms);
+    zmk_behavior_queue_add(event.position, data->binding, false, 0);
+    reset_timer(data, event);
+}
+
+static int on_keymap_binding_pressed(struct zmk_behavior_binding *binding,
+                                     struct zmk_behavior_binding_event event) {
+    const struct device *dev = device_get_binding(binding->behavior_dev);
+    const struct behavior_turbo_config *cfg = dev->config;
+    struct behavior_turbo_data *data = dev->data;
+
+    if (!data->is_active) {
+        data->is_active = true;
+
+        LOG_DBG("%d started new turbo", event.position);
+        data->press_time = k_uptime_get();
+        k_work_init_delayable(&data->release_timer, behavior_turbo_timer_handler);
+        zmk_behavior_queue_add(event.position, cfg->binding, true, cfg->tap_ms);
+        zmk_behavior_queue_add(event.position, cfg->binding, false, 0);
+        reset_timer(data, event);
+    } else {
+        clear_turbo(data);
+    }
+    return ZMK_BEHAVIOR_OPAQUE;
+}
+
+static int on_keymap_binding_released(struct zmk_behavior_binding *binding,
+                                      struct zmk_behavior_binding_event event) {
+    const struct device *dev = device_get_binding(binding->behavior_dev);
+    const struct behavior_turbo_config *cfg = dev->config;
+    struct behavior_turbo_data *data = dev->data;
+
+    if (data->is_active) {
+        data->is_pressed = false;
+        int32_t elapsedTime = k_uptime_get() - data->press_time;
+        LOG_DBG("turbo elapsed time: %d", elapsedTime);
+        if (elapsedTime > cfg->toggle_term_ms) {
+            clear_turbo(data);
+        }
+    }
+    return 0;
+}
+
+static const struct behavior_driver_api behavior_turbo_key_driver_api = {
+    .binding_pressed = on_keymap_binding_pressed,
+    .binding_released = on_keymap_binding_released,
+};
+
+#define _TRANSFORM_ENTRY(idx, node)                                                                \
+    {                                                                                              \
+        .behavior_dev = DT_LABEL(DT_INST_PHANDLE_BY_IDX(node, bindings, idx)),                     \
```

Comment:
Needs to be refactored to:

```suggestion
        .behavior_dev = DT_PROP(DT_INST_PHANDLE_BY_IDX(node, bindings, idx), label),                     \
```

For some Zephyr changes.